docs/cave/d2d-gis-shapefiles.md

```diff
@@ -22,6 +24,8 @@ Using the **Preferences** dialog, you can modify certain GIS display attributes
 
 The attributes that can be modified within CAVE apply to the map/map product image displayed on the Main Display Pane. They are: color of the highlights; highlight line style; highlight line width; and map product opacity. The button next to the name of the attribute displays a swatch of the current value. Selecting this button opens a dialog for modifying that attribute and redefining the default value.
```

Comment:
I think the list of attributes that can be modified should be comma separated, not semi colon.  And I would probably like to see them as links to sub sections.   Meaning the bulleted list below should actually be changed to `###` headers.